apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -52,6 +51,8 @@ public sealed class PooledSession : IDisposable
         /// </summary>
         public SessionName SessionName { get; }
 
+        internal Session Session { get; }
```

Comment:
Doc comment would be handy here - along with the relationship to `SessionName`. (Can we make SessionName just delegate to Session.SessionName?)